doctrine-phpcr-odm-mapping.xsd

```diff
@@ -44,6 +44,10 @@
         <xs:anyAttribute namespace="##other"/>
     </xs:complexType>
 
+    <xs:complexType name="child-class">
+        <xs:attribute name="name" type="xs:string"/>
```

Comment:
@dbu changed to an attribute as there are no examples of node-value elements in Doctrine. /cc @WouterJ